src/org/labkey/test/components/domain/DomainFieldRow.java

```diff
@@ -660,6 +661,12 @@ protected ElementCache newElementCache()
         return new ElementCache();
     }
 
+    public DomainFieldRow setSampleTypeLookup(String sampleTypeName)
+    {
+        elementCache().getSampleTypeSelect().selectByVisibleText(sampleTypeName);
+        return this;
+    }
+
```

Comment:
Haha I have the same change in my branch for  SourcesCRUD. The only difference I have is I have a getter and I explicitly call expand. Here are my changes:

`    public String getSampleType()
    {
        expand();
        return elementCache().getLookupSampleTypeSelect().getFirstSelectedOption().getText();
    }

    public DomainFieldRow setSampleType(String sampleTypeName)
    {
        expand();
        elementCache().getLookupSampleTypeSelect().selectByVisibleText(sampleTypeName);
        return this;
    }
`